ci/pipeline-dev.yml

```diff
@@ -386,7 +268,7 @@ resources:
     source:
       repository: ((pages-repository-path))
       access_token: ((gh-access-token))
-      base_branch: ((git-branch))
+      base_branch: main
```

Comment:
Should we set this to `staging` until we make the switch to a single branch and tag deployment? Or should we update this PR to point to `main`?

Reply:
Yeah let's update this PR to point to main (even if the tests here aren't super meaningful against this code, we'll at least see the dev deploy try to run)